maufbapi/proxy.py

```diff
@@ -1,13 +1,17 @@
+from typing import Optional
 import json
 import logging
 import urllib.request
 
 
 class ProxyHandler:
-    current_proxy_url: str | None = None
+    current_proxy_url: Optional[str] = None
```

Comment:
`| None` is the correct way, just need to `from __future__ import annotations`

(the only exception is files using `SerializableAttrs`, which depends on being able to parse type hints at runtime)

Reply:
Ah! Keep getting caught out by this I even reverted from it 😢 rewritten in https://github.com/mautrix/facebook/pull/239/commits/e5e6ca9568e857603c4568314279a8a36f5ab8b1